### ISSUE TITLE
Wrong/new location for fan info

### ISSUE THREAD

**Issue by FlyingLemming** [ebaauw/homebridge-rpi#178]
On my Raspi 5 with a brand-new installation of Bookworm 64bit, the directory for getting the fan speed info is /sys/devices/platform/cooling_fan/hwmon/hwmon**3** instead of .../hwmon**2**.

No idea about other hardware and OS versions.

**Comment by FlyingLemming** [comment]
Good point. However, I just restarted the Pi three times and the path remains `.../hwmon3`.

I am using the stand-alone fan (["Raspberry Pi Active Cooler"](https://www.raspberrypi.com/products/active-cooler/)) without any further modifications to it. 

In addition, I have a Geekworm X1001 NVMe Shield installed as well as a Geekworm X1200 UPS Shield. Thinking of that – I recently transferred the OS from the SD card to a NVMe SSD. Might that be the cause...? 🤔

I am only using the software locally. For the time being, I just modfied the path within my `RpiInfo.js` and it works fine.
